src/worker/worker/tests/testdata.py

```diff
@@ -117,62 +119,60 @@
ert.\nProvide a name for the alert and provide the interval at which the alert should be triggered.\nSelect the alert action as "Send to Microsoft Azure Data Explorer".\n\n\xa0\n\n\xa0\n5. Configure the Azure Data Explorer connection details such as application client Id, application client secret, cluster name, database name, table name.\n\n\xa0\n\xa06.\xa0When the alert is created, it should be visible in Splunk Dashboard -> Alerts\n\n\xa0\nStep 4: Verify data in Azure Data Explorer\n\xa0\n\nStart monitoring the Azure Data Explorer logs to ensure proper data ingestion.\nOnce the alert is triggered in Splunk, the data will be ingested to Azure Data Explorer.\nVerify the data in Azure Data Explorer using the database and table name in the previous step.\n\n\n\xa0\nAzure Data Explorer Addon Parameters\n\xa0\nThe following is the list of parameters which need to be entered/selected while configuring the addon:\n\nAzure Cluster Ingestion URL: Represents the ingestion URL of the Azure Data Explorer cluster in the ADX portal.\nAzure Application Client Id: Represents the Azure Application Client Id credentials required to access the Azure Data Explorer cluster.\nAzure Application Client secret: Represents the Azure Application Client secret credentials required to access the Azure Data Explorer cluster.\nAzure Application Tenant Id: Represents the Azure Application Tenant Id required to access the Azure Data Explorer cluster.\nAzure Data Explorer Database Name: This represents the name of the database created in the Azure Data Explorer cluster, where we want our data to be ingested.\nAzure Data Explorer Table Name: This represents the name of the table inside the database created in the Azure Data Explorer cluster, where we want our data to be ingested.\nAzure Data Explorer Table Mapping Name: This represents the Azure Data Explorer table mapping used to map to the column of created Azure Data Explorer table.\nRemove Extra Fields: This represents whether we want to remove empty fields in the splunk event payload\nDurable Mode: This property specifies whether durability mode is required during ingestion. When set to true, the ingestion throughput is impacted.\n\n\xa0\nPlease refer to the following links for further details:Link to Splunk Base Addon :\xa0Microsoft Azure Data Explorer Add-On for Splunk | Splunkbase\nLink to Github Source code of Addon:\xa0azure-kusto-splunk/splunk-adx-alert-addon at main · Azure/azure-kusto-splunk (github.com)\n\xa0\nConclusion\n\xa0\nIn this blog post, we have seen how Microsoft Azure Data Explorer Add-On for Splunk can help us ingest data from Splunk to Azure Data Explorer, a powerful data analytics platform. We have also learned about the types of data that are most suitable for Azure Data Explorer, and how to configure the add-on and use alert actions to send data to Azure Data Explorer. By using this add-on, we can leverage the benefits of both Splunk and Azure Data Explorer, and gain deeper insights from our data.',
```

Comment:
@stefaniasava is there a reason that the content attribute is not present anymore?

Reply:
I added it back